stockflux-securities-master/src/components/AddSecurityButton.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "./AddSecurityButton.css";
+
+const AddSecuritiesButton = () => {
+  return (
+    <Link to="/inputform/">
```

Comment:
Can we remove the trailing slash here to match the defined route?

Reply:
yep gotten rid of trailing slashes